Update plus-addressing-in-exchange-online.md (#2342)
MicrosoftDocs/OfficeDocs-Exchange: Exchange/ExchangeOnline/recipients-in-exchange-online/plus-addressing-in-exchange-online.md

An SMTP email address uses the basic syntax: `<local-part>@<domain>`. For example, sean@contoso.com. 

Plus addressing uses the syntax: `<local-part>+<tag>@<domain>`. For example, sean+newsletter@contoso.com. 

The original email address must be valid; the `+tag` value that you add is arbitrary, although regular character restrictions for SMTP email addresses apply (for example, no spaces). For more information about using plus addresses, see **Using plus addresses**.

By default, plus addressing support is disabled in Exchange Online. Since Exchange Online has always supported regular email addresses that already contain the plus sign, if you enable plus addressing, these email addresses might stop working. See the note below in **Enable plus addressing in your Exchange Online organization**.

You can’t enable plus addressing in the Exchange admin center (EAC); you can only enable it through Exchange Online PowerShell. If your organization’s emails are routed through Exchange Online to your on-premises servers, mailboxes hosted on-premises will also be able to use plus addresses.  

> >[!NOTE]

Reviewer: ```suggestion
> [!NOTE]
```

> > Currently, plus addresses are only supported for mailboxes but support for Groups and DLs is being rolled out and will be available by the end of December 2020. 

Reviewer: ```suggestion
> Currently, plus addresses are only supported for mailboxes but support for Groups and distribution lists is being rolled out and will be available by the end of December 2020. 
```

   Set-OrganizationConfig -AllowPlusAddressInRecipients $true
   ```

> >[!NOTE]

Reviewer: ```suggestion
> [!NOTE]
```

> > This feature was rolled out behind a setting because, historically, customers have been able to use plusses in addresses for mailboxes in Exchange Online and on-premises servers. When this feature is enabled, Exchange Online will first check if the full address can resolve to a mailbox that the service is aware of. It is only when that resolution fails, that a plus is looked for and a second attempt to resolve the address without the plus and tag is done. This means that the feature is compatible with addresses containing plusses that Exchange Online knows about. If you relay messages to a mailbox on-premises that does not resolve in Exchange Online, message delivery will be affected. The messages will be parsed and addressed to the parsed address e.g. sean@contoso.com instead of the full sean+newsletter@contoso.com, using the example above.  

Reviewer: ```suggestion
> This feature was rolled out behind a setting because, historically, customers have been able to use plusses in addresses for mailboxes in Exchange Online and on-premises servers. When this feature is enabled, Exchange Online will first check if the full address can resolve to a mailbox that the service is aware of. It is only when that resolution fails that a plus is looked for and a second attempt to resolve the address without the plus and tag is done. This means that the feature is compatible with addresses containing plusses that Exchange Online knows about. If you relay messages to a mailbox on-premises that does not resolve in Exchange Online, message delivery will be affected. The messages will be parsed and addressed to the parsed address, e.g. sean@contoso.com instead of the full sean+newsletter@contoso.com, using the example above.  
```
